xarray/backends/netCDF4_.py

```diff
@@ -495,6 +494,7 @@ def prepare_variable(
             )
 
         for k, v in attrs.items():
+            # TODO: update now that we dont support this old version of netCDF4
             # set attributes one-by-one since netCDF4<1.0.10 can't handle
             # OrderedDict as the input to setncatts
             _set_nc_attribute(nc4_var, k, v)
```

Comment:
I think we can update this now. @shoyer - do you recall what the setter syntax is for a dict of attributes?